src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ interface Item {
   [key: string]: any;
 }
 
-export function groupByKey(items, field) {
+export function groupByKey(items: Item[], field: keyof Item): Item {
   // write code here;
+  const result: Item = {};
```

Comment:
The result should be not of an 'Item' type. Each key should be an array of 'items' (not 'any')

Create the interface
```
{
  [key: string] : Item[],
}
```
and use it as a result type

Reply:
I've just fixed it. Thank you :)